data/libs/systems/vip.lua

```diff
@@ -52,16 +53,15 @@ end
 
 function CheckPremiumAndPrint(player, msgType)
 	if player:getVipDays() == 0xFFFF then
-		player:sendTextMessage(msgType, "You have infinite amount of VIP days left.")
+		player:sendTextMessage(msgType, config.vipDaysInfiniteMessage)
 		return true
 	end
 
 	local playerVipTime = player:getVipTime()
 	if playerVipTime < os.time() then
-		local msg = "You do not have VIP on your account."
-		player:sendTextMessage(msgType, msg)
+		player:sendTextMessage(msgType, config.noVipStatusMessage)
```

Comment:
As this message is only displayed once, I think it needs to be used in a variable.